lunatrace/bsl/hasura/migrations/lunatrace/1664218225933_release_analysis_result/up.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TYPE finding_state AS ENUM (
+    'vulnerable',
+    'not_vulnerable'
+    );
+
+CREATE TABLE package.release_analysis_result
```

Comment:
I'm not convinced this is a "release_analysis". I don't see a release_id. 

I see a manifest_dependency_edge_id, and since we made that name such a mouthful, seems like we're now avoiding trying to use it.  Maybe this is just a "false_positive_analysis", I'd be fine with that.